go.mod

```diff
@@ -40,7 +40,7 @@ require (
 	github.com/tangzero/inflector v1.0.0
 	github.com/withfig/autocomplete-tools/packages/cobra v1.2.0
 	go.mongodb.org/atlas v0.30.0
-	go.mongodb.org/atlas-sdk v0.15.0
+	go.mongodb.org/atlas-sdk/v20230201001 v20230201001.0.1-0.20230704132156-6560cf3b3bce
```

Comment:
Q. Shouldn't this just be `go.mongodb.org/atlas-sdk/v20230201001.0.1`?

Reply:
I forgot to explain it. 
This is using an unreleased version. 
Once I merge I want to verify if dependabot is going to pick up next major release of the SDK.
I need to merge this with unrelased version and will replace it almost instantly with released one.